docs/konnectors.md

```diff
@@ -320,18 +320,40 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
+The konnectors will be installed in the .cozy_konnectors directory (not in VFS ?) using npm install
+(with the ```--ignore-scripts``` option). This command will give the possibility to install konnectors
+from npm, github, gitlab, bitbucket, tar.gz url. The stack will have to check that the given package
+does not use a "file:" path.
```

Comment:
For a konnector, do we have one install for all users, one install per user? If one installs for all users, how do we differentiate the same konnector installed from two source? For example, if user `toto` asks to use the trainline konnector from npm, and user `titi` asks to install the tranline konnector too, but from git